src/components/pdf/SkillItem.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, View } from '@react-pdf/renderer';
+import * as pdfStyles from '~/components/pdf/pdf.styles.js';
+
+/**
+ * Create a single skill Item.
+ */
+const SkillItem = ({ label, list }) => {
+  return (
+    <View style={pdfStyles.skills.content}>
+      <Text>{`${label} `}</Text>
+      <SubSkillItem subSkills={list} />
+    </View>
+  );
+};
+
+const SubSkillItem = ({ subSkills }) => {
+  if (subSkills.length < 1) {
+    return (
+      <>
```

Comment:
`<></>` isn't required as we are only returning <Text> component

Reply:
done